### PR TITLE
Iss39 active learning

### PR DESCRIPTION
**Pull request by michellebieger**
Thank you for your help Alexander!

Features new candidate_points_campaign implementing the active learning from the backend. 

Also includes more detailed error messaging on the client-side.

### CODE REVIEW

File: scripts/api/get_candidate_points_model.py
Comment (alexander-mead) [inline scripts/api/get_candidate_points_model.py:1]: Looks good!

**Review by alexander-mead — CHANGES_REQUESTED** [review]

**Review by alexander-mead — APPROVED** [review]

**Comment by alexander-mead** [comment]
This is good to go from my point of view. @FreddyWordingham I won't merge, but you should feel free to.

**Comment by FreddyWordingham** [comment]
Everything working on my branch and in monolith

One note: `api_key` needs to be `api-key` (the AWS HTTP Gateway changes `-` to `_` -.- )

**Comment by alexander-mead** [comment]
Thanks API gateway, thanks. 😀

**Pull request merged**